wp-content/themes/oceanwp-child/comments.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * The template for displaying Comments.
+ *
+ * The area of the page that contains both current comments and the comment
+ * form. The actual display of comments is handled by a callback to
+ * oceanwp_comment() which is located at functions/comments-callback.php
+ *
+ * @package OceanWP WordPress theme
+ */
+
+// Return if password is required.
+if ( post_password_required() ) {
+	return;
+}
+
+// Add classes to the comments main wrapper.
+$classes = 'comments-area clr';
+
+if ( get_comments_number() !== 0 ) {
+	$classes .= ' has-comments';
+}
+
+if ( ! comments_open() && get_comments_number() < 1 ) {
+	$classes .= ' empty-closed-comments';
+	return;
+}
+
+if ( 'full-screen' === oceanwp_post_layout() ) {
+	$classes .= ' container';
+}
+
+// Get comment form position.
+$comment_position = apply_filters( 'ocean_comment_form_position', get_theme_mod( 'ocean_comment_form_position' ) );
+$comment_position = $comment_position ? $comment_position : 'before';
+
+// Comment form args.
+$args = array(
+	/* translators: 1: logged in to comment */
+	'must_log_in'          => '<p class="must-log-in">' . sprintf( esc_html__( 'You must be %1$slogged in%2$s to post a comment.', 'oceanwp' ), '<a href="' . wp_login_url( apply_filters( 'the_permalink', get_permalink() ) ) . '">', '</a>' ) . '</p>',
+	'logged_in_as'         => '<p class="logged-in-as">' . esc_html__( 'Logged in as', 'oceanwp' ) . ' <a href="' . admin_url( 'profile.php' ) . '">' . $user_identity . '</a>.<span class="screen-reader-text">' . oceanwp_theme_strings( 'owp-string-comment-profile-edit', false, 'oceanwp' ) . '</span> <a href="' . wp_logout_url( get_permalink() ) . '" aria-label="' . oceanwp_theme_strings( 'owp-string-comment-logout-text', false, 'oceanwp' ) . '">' . esc_html__( 'Log out &raquo;', 'oceanwp' ) . '</a></p>',
+	'comment_notes_before' => false,
+	'comment_notes_after'  => false,
+	'comment_field'        => '<div class="comment-textarea"><label for="comment" class="screen-reader-text">' . esc_html__( 'Comment', 'oceanwp' ) . '</label><textarea name="comment" id="comment" cols="39" rows="4" tabindex="0" class="textarea-comment" placeholder="' . oceanwp_theme_strings( 'owp-string-comment-placeholder', false, 'oceanwp' ) . '"></textarea></div>',
```

Comment:
Because this is generating multiple fields with the same id, we should remove the id and wrap the text area with the label:
```suggestion
		'comment_field'        => '<div class="comment-textarea"><label for="comment"><span class="screen-reader-text">' . esc_html__( 'Comment', 'oceanwp' ) . '</span><textarea name="comment" cols="39" rows="4" tabindex="0" class="textarea-comment" placeholder="' . oceanwp_theme_strings( 'owp-string-comment-placeholder', false, 'oceanwp' ) . '"></textarea></label></div>',
```